src/pymortests/demos.py

```diff
@@ -13,7 +13,9 @@
 from pymortests.base import runmodule, check_results
 from pymor.core.exceptions import PySideMissing
 from pymor.gui.qt import stop_gui_processes
+from pymor.core.config import is_windows_platform
 
+pytestmark = pytest.mark.skipif(is_windows_platform(), reason='interpreter crashes')
```

Comment:
Probably related to the GUI? In that case, it might also be an option to only disable the GUI output with some nice monkey patching?